src/Utils.js

```diff
@@ -17,6 +17,13 @@ export function propsOfNode(node) {
   if (node && node._reactInternalComponent && node._reactInternalComponent._currentElement) {
     return (node._reactInternalComponent._currentElement.props) || {};
   }
+  if (REACT15 && node) {
+    const instanceKey = Object.keys(node).find(key => key.match(/^__reactInternalInstance\$/));
```

Comment:
since we're doing this in more than one place, it's probably best if we pull this out into a helper method that also makes it more semantically clear.